src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -228,10 +226,27 @@ public boolean init(Expression<?>[] exprs,
 			parseIfPassed = true;
 		}
 
-		Kleenean hadDelayBefore = getParser().getHasDelayBefore();
+		Kleenean hadDelayBefore = parser.getHasDelayBefore();
 		if (!multiline || type == ConditionalType.THEN)
 			loadCode(sectionNode);
-		hasDelayAfter = getParser().getHasDelayBefore();
+
+		if (type == ConditionalType.ELSE) {
+			List<SecConditional> conditionals = getPrecedingConditionals(triggerItems);
+			conditionals.add(0, this);
+			for (SecConditional conditional : conditionals) {
+				if (conditional.multiline && conditional.type != ConditionalType.THEN)
+					continue;
+				ExecutionIntent triggerIntent = conditional.triggerExecutionIntent();
+				if (triggerIntent == null) {
+					executionIntent = null;
+					break;
+				}
+                if (executionIntent == null || triggerIntent.compareTo(executionIntent) < 0)
+                    executionIntent = triggerIntent;
```

Comment:
some comments explaining the purpose of this code would be nice, it's pretty dense